lib/trino-orc/src/main/java/io/trino/orc/metadata/Footer.java

```diff
@@ -93,6 +96,11 @@ public Optional<Integer> getWriterId()
         return writerId;
     }
 
+    public Optional<CalendarKind> getCalendar()
```

Comment:
What's the purpose of this. It seems unused in Trino.

Separately, if Trino only supports prolectic gregorian, it should throw an error or do the appropriate conversions when it encounters a different calendar while reading a file.

Reply:
> What's the purpose of this? 

This is to ensure that other engines like Hive can be forced to read the data in proleptic gregorian. 

As a follow up we are thinking on do a conversion when we encounter a different calendar when reading a file.

Reply:
If `Optional.empty()` means Julian calendar, then the reader should return that instead. I don't see the benefit of signaling that the calendar type is missing.